deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -251,8 +319,34 @@ spec:
               description: EnvVarPrefix is the prefix for environment variables
               type: string
             mountPathPrefix:
-              description: MountPathPrefix is the prefix for volume mount
+              description: "Important: Run \"operator-sdk generate k8s\" to regenerate
```

Comment:
description here should not change, the previous one looks good ` MountPathPrefix is the prefix for volume mount`